src/three/TilesRenderer.js

```diff
@@ -661,10 +662,15 @@ export class TilesRenderer extends TilesRendererBase {
 					// (such as applying RTC_CENTER) meaning they should happen _after_ the z-up
 					// rotation fix which is why "multiply" happens here.
 					const { scene } = result;
-					scene.updateMatrix();
-					scene.matrix
-						.multiply( upRotationMatrix )
-						.decompose( scene.position, scene.quaternion, scene.scale );
+
+					if ( scene ) {
```

Comment:
Thanks - I think it will be easier to fix this if we just assign "scene" here if it's not present:

```js
result.scene = result.scene || new Group();
```

No other changes should be needed I don't think.